tests/suites/test_suite_pkcs1_v21.function

```diff
@@ -182,6 +182,59 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void pkcs1_rsassa_pss_sign_ext( int mod, int radix_P, char * input_P, int radix_Q,
+                                char * input_Q, int radix_N, char * input_N,
+                                int radix_E, char * input_E, int digest, int hash,
+                                data_t * message_str, data_t * rnd_buf,
+                                data_t * result_str, int fixed_salt_length,
+                                int result )
+{
+    unsigned char hash_result[MBEDTLS_MD_MAX_SIZE];
+    unsigned char output[512];
+    mbedtls_rsa_context ctx;
+    mbedtls_test_rnd_buf_info info;
+    mbedtls_mpi N, P, Q, E;
+
+    info.buf = rnd_buf->x;
+    info.length = rnd_buf->len;
+
+    mbedtls_mpi_init( &N ); mbedtls_mpi_init( &P );
+    mbedtls_mpi_init( &Q ); mbedtls_mpi_init( &E );
+    mbedtls_rsa_init( &ctx, MBEDTLS_RSA_PKCS_V21, hash );
+
+    memset( hash_result, 0x00, sizeof( hash_result ) );
+    memset( output, 0x00, sizeof( output ) );
+
+    TEST_ASSERT( mbedtls_mpi_read_string( &P, radix_P, input_P ) == 0 );
+    TEST_ASSERT( mbedtls_mpi_read_string( &Q, radix_Q, input_Q ) == 0 );
+    TEST_ASSERT( mbedtls_mpi_read_string( &N, radix_N, input_N ) == 0 );
+    TEST_ASSERT( mbedtls_mpi_read_string( &E, radix_E, input_E ) == 0 );
```

Comment:
Pre-existing, optional: it would be more efficient to remove the `radix_` parameters, turn the `input_` parameters into `data_t *` and use `mbedtls_mpi_read_binary()` here. (Concretely, this would reduce the size of the generated test binary by embedding the binary version of the parameters rather than the hex version, which may matter for testing on constrained platforms.)

Since the existing tests already have this issue, I won't block this PR on that, but if you feel like improving it, that would be appreciated.

Reply:
Done